lerna.json

```diff
@@ -7,7 +7,8 @@
         "DEPENDENCIES.md",
         "terra-application-name",
         "terra-application-utility",
-        "terra-embedded-component"
+        "terra-embedded-component",
+        "terra-aggregator"
```

Comment:
Is this something we want to keep in the lerna ignore config or is this ready to be released soon?